modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -0,0 +1,128 @@
+= Migrating User Key-based Encryption to Master Key-based Encryption
+
```

Comment:
Post the headline (= Migrating...), valid for ALL documents (except very short once or without subsections), add the following: `:toc: right`. This creates an automatic created table of contents.

Create a subsection named `== Introduction` where you describe briefly what it is about.